src/FsAutoComplete.Core/Commands.fs

```diff
@@ -731,10 +731,10 @@ module Commands =
 
   let symbolUseWorkspaceAux
     (getDeclarationLocation: FSharpSymbolUse * IFSACSourceText -> Async<SymbolDeclarationLocation option>)
-    (findReferencesForSymbolInFile: (string<LocalPath> * FSharpProjectOptions * FSharpSymbol) -> Async<Range seq>)
+    (findReferencesForSymbolInFile: (string<LocalPath> * FSharpProjectSnapshot * FSharpSymbol) -> Async<Range seq>)
```

Comment:
You'll see lots of mechanical changes from `FSharpProjectOptions` -> `FSharpProjectSnapshot`. Which include changes from Array to List in various places.